docs/user/commercial/sharing.rst

```diff
@@ -67,6 +67,13 @@ view your project.
    This is useful for when you have documentation you want users to bookmark.
    They can enter a URL directly and enter the password when prompted.
 
+.. warning::
+
+   Since multiple passwords can be created for a single project,
+   each password must be unique within that project.
+   If you generate multiple sharing links using the same password for a project,
+   only the first link will work.
```

Comment:
Can we check against this when we do the hashing, or will they hash differently because of the salt?

Reply:
We can check, but we will need to check each hashed password, which will be slow if users have lots of passwords, hopefully it's more fast with the new hash algo we are going to implement. So, we could have that feature.

Reply:
The form is now able to check for dups with https://github.com/readthedocs/readthedocs-corporate/pull/1849, should we still mention this warning?

Reply:
I think we should be fine closing this, if the form can check for it.